portfolio/src/main/webapp/style.css

```diff
@@ -58,13 +58,16 @@ li{
     padding-left: 25px;
     font-family: Barlow;
 }
+.bottom{
+    padding-left: 100px
+}
 
 
 
 #education {padding-top:50px;height:500px;color: #fff; background-color: rgba(224, 224, 82, 0.137);}
 #work {padding-top:50px;height:500px;color: #fff; background-color: rgba(117, 187, 148, 0.267);}
-#projects {padding-top:50px;height:500px;color: #fff; background-color: #e4b67242;}
+#projects {padding-top:50px;padding-bottom:1500px; height:500px;color: #fff; background-color: #e4b67242;}
```

Comment:
That's a massive bottom padding. This seems like something that should be handled a different way. Is this just something trying to fill the screen?

Reply:
Yes. I am trying to fit the two images that I put under projects. I put the padding because before it was overlapping with the other sections. Is there another way to fix this?

Reply:
The short answer is - 'yes, there's another way'

I'd have to take a look at it thought to give proper advice on the subject. Padding-bottom affects the distance from the bottom of an element's contents to the top of the element's border - https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Box_Model/Introduction_to_the_CSS_box_model

What this style rule is doing is putting 1500px of space below the contents of #projects. This may have the desired effect but is likely not what you actually want to do. If you haven't tried it, I'd recommend opening Chrome Dev tools (assuming you're using chrome) by right clicking on the #projects section and clicking "Inspect Element". Not only will this show you what styles are applied, you'll be able to change them and play around with them on the fly. It's very helpful to see how things are affected without having to make changes and redeploy constantly. Just remember that any changes you make only apply to the currently loaded page in chrome and will disappear if the page is re-rendered.